prisma/migrations/20241104165143_add/migration.sql

```diff
@@ -0,0 +1,31 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `name` on the `user` table. All the data in the column will be lost.
+  - You are about to drop the `post` table. If the table is not empty, all the data it contains will be lost.
```

Comment:
**issue**: Dropping the `post` table will result in data loss. Confirm that this is intended and that any important data has been backed up or migrated.